cmd/get_client_certificate_test.go

```diff
@@ -51,14 +54,22 @@ var _ = Describe("GetClientCertificate", func() {
 		errOut    *util.SafeBytesBuffer
 		out       *util.SafeBytesBuffer
 
-		shootCaData []byte
-		ec          v1beta1.ExecCredential
+		shootCaData           []byte
+		v1beta1ExecCredential clientauthenticationv1beta1.ExecCredential
+		v1ExecCredential      clientauthenticationv1.ExecCredential
 	)
 
 	BeforeEach(func() {
-		scheme := runtime.NewScheme()
+		scheme := clientgoscheme.Scheme
```

Comment:
In gardenctl we do this scheme initialization in the init function of the `..._suite_test.go` file.

Reply:
✅